compose/service.py

```diff
@@ -885,17 +885,19 @@ def has_host_port(binding):
 
         return any(has_host_port(binding) for binding in self.options.get('ports', []))
 
-    def pull(self, ignore_pull_failures=False):
+    def pull(self, ignore_pull_failures=False, silent=False):
```

Comment:
There are a couple problems with doing this:

First, `parallel_execute` outputs lines very similar to `Service.pull`. It's

`Pulling service (image:tag) ...`

vs.

`Pulling service ...`

So we get the same thing output twice for each service, basically.

Second, `parallel_execute` outputs some terminal commands to go back and update older lines, and if anything else outputs something it messes up the terminal commands and the wrong lines are updated.